shared/middleware/net_sim_api.rb

```diff
@@ -505,8 +506,18 @@ def delete_messages_for_nodes(shard_id, node_ids)
   # @param [Number] value (default 1) value of measurement, omit if we only care
   #   about event counts.
   def record_metric(event_type, value = 1)
```

Comment:
I can't comment on line 502, but we should update that comment to say CloudWatch